.travis.yml

```diff
@@ -42,7 +42,18 @@ install:
       protobuf-c
       thrift
   - brew outdated libyaml || brew upgrade libyaml
-  - brew outdated json-c  || brew upgrade json-c
+  - git clone https://github.com/json-c/json-c.git
+  - cd json-c
+  - sh autogen.sh
+  - git reset --hard json-c-0.12.1-20160607
+  - git log
+  - git reset json-c-0.12.1-20160607 --hard
```

Comment:
why there are duplicate lines of this? Suggest to use `git checkout json-c-0.12.1-20160607` instead of reset.